examples/bs4.R

```diff
@@ -78,7 +78,7 @@ shiny::runApp(list(
     # or below if initialized without default data source
     # coh <- cohortBuilder::cohort()
 
-    shinyCohortBuilder::cb_server(id = "fk", coh, feedback = TRUE)
+    shinyCohortBuilder::cb_server(id = "fk", run_button = "global", coh, feedback = TRUE)
```

Comment:
`coh` should be the second param for `cb_server`

Reply:
`run_button = "global"` is actually named argument so this will work, but I'll do it for readability imrpvement.
Thank you for noting that.